index.bs

```diff
@@ -579,16 +571,14 @@ When this method is invoked, the user agent MUST execute the following algorithm
             {{CredentialsContainer/create()}}.
     </li>
 
-1. Let |normalizedParameters| be a new [=list=] whose [=list/items=] are pairs of {{PublicKeyCredentialType}} and a
-    [=dictionary=] type (as returned by [=normalizing an algorithm=]).
+1. Let |normalizedParameters| be a new [=list=] whose [=list/items=] are pairs of {{PublicKeyCredentialType}} and
+    an {{AlgorithmIdentifier}}.
```

Comment:
the type `{{AlgorithmIdentifier}}` is now a dangling pointer in that it is not itself defined.